Dockerfile

```diff
@@ -32,23 +55,18 @@ RUN curl -fsSLO "$RCON_URL" \
     && tar xfz rcon-0.10.3-amd64_linux.tar.gz \
     && chmod +x "rcon-0.10.3-amd64_linux/$RCON_BINARY" \
     && mv "rcon-0.10.3-amd64_linux/$RCON_BINARY" "/usr/local/bin/${RCON_BINARY}" \
```

Comment:
Removed symbolic link makes sense.
RCONCLI needs good documentation that its for internal usage, so the log is better